e2e/tests/relay.feature

```diff
@@ -19,6 +19,19 @@ Feature: Relay Namespace
     #    When the application "app2" sends the supplier "supplier1" a successful request for service "rest" with path "/quote"
     #    Then a "tokenomics" module "ClaimSettled" end block event is broadcast
 
+    Scenario: App can send WEBSOCKETS relays to Supplier
+        Given the user has the pocketd binary installed
+        And the application "app3" is staked for service "anvilws"
+        And the supplier "supplier1" is staked for service "anvilws"
+        And the session for application "app3" and service "anvilws" contains the supplier "supplier1"
+        # Establishing the websockets connection should be done when a session starts.
+        # This is to maximize the number of responses before the session ends and the
+        # websockets connection is closed.
+        And the application "app3" establishes a websockets connection for service "anvilws"
+        Then the user receives ethereum subscription events
```

Comment:
Can we do something like:

```feature
Then the application "app3" subscribes for one "eth_subscription" event
And the supplier sends "N" "eth_buscription" to the application "app3"
```

I'm not sure:
- Who sends
- who subscribes
- Who is the user?
- How many events are expected?

Reply:
Updated with:
```
Then the application receives EVM subscription events until the session ends
And the subscription is closed before claim window open height is reached
```

* In `... the application "app3"...` there is no use for `app3`, that's why it's omitted.
* It is subscribing for as many events as `anvilws` is willing to send until the `RelayMiner` closes the connection due to session end.
* The number of messages received depends on the session duration and the `anvil` block time.
* We can observe a single event emission but we also want to observe that the connection gets closed when the session ends.